### ISSUE TITLE
unable to upload changes (status 400, Placeholder was not found for reference)

### ISSUE THREAD

**Issue by dieterdreist** [bryceco/GoMap#816]
I got this error on upload:
![IMG_3103](https://github.com/user-attachments/assets/3ff15aa9-c5ab-4d3c-8b98-618bb2b2f609)

When I try to open the OSC in Josm, I get: SEVERE: Error - <html>Could not read file 'osmChange.osc'.<br>Error is:<br>Relation with external id '31192' refers to a missing primitive with external id '-1433831'.</html>

**Comment by dieterdreist** [comment]
i solved it by removing the missing reference, but somehow it must have happened (I think it may have come from a way-split, where the road was part of the route, and there was probably some hicup with the mobile network on upload, as I did it from a driving car in a rural area, but I wanted to let you know about it in case it is something more systematic)

**Comment by bryceco** [comment]
I agree it sounds like you split a way that belonged to a relation, and then something happened to the way. Probably not a network issue. Thanks for your report!
